client/src/components/AddContactModal.jsx

```diff
@@ -33,9 +34,32 @@ const AddContactModal = ({ setContacts }) => {
   });
 
   const toast = useToast();
+  const [errors, setErrors] = useState({});
+
+  const validate = () => {
+    const newErrors = {};
+    if (!formData.name || formData.name.trim().length < 3) {
```

Comment:
Please use the same `contactValidation` utility function here that you used in the EditContactModal component. Thanks, I appreciate the efforts.

Reply:
I have updated the code to use the same contactValidation utility function as in the EditContactModal component. Please verify and let me know if any further changes are needed. Thanks for your feedback!.